### PR TITLE
Changed provisioning form. Added AngularSelect for v5.5

### PR DESCRIPTION
**Pull request by otsuman**
{{pytest: cfme/tests/cloud/test_provisioning.py -k test_provision_from_template --use-provider rhos7-ga}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 46b466197f8e4002e612521a56244c30cae333cf:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/provisioning.py [hunk 1]
```diff
@@ -123,32 +166,46 @@ def select_security_group(sg):
...
-            ('instance_type', ui.Select('select#hardware__instance_type')),
-            ('guest_keypair', ui.Select('select#hardware__guest_access_key_pair',
+            ('instance_type', {
+                version.LOWEST: ui.Select('select#hardware__instance_type'),
+                '5.5': AngularSelect('hardware__instance_type')}),
+            ('guest_keypair', {
+                version.LOWEST: ui.Select('select#hardware__guest_access_key_pair',
+                none={'5.4': "<None>",
+                      version.LOWEST: "<No Choices Available>"}),
+                '5.5': AngularSelect('hardware__guest_access_key_pair',
```
Comment (psav) [inline cfme/provisioning.py]: Indentation should match

**Comment by dajoRH** [comment]
Lint report for commit d640a64b0ad94a3fdcb583cfa92380ec2895f787:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
Can we get a test run here on PRT that is using the file modifications?

**Comment by otsuman** [comment]
{{pytest: cfme/tests/cloud/test_provisioning.py -k test_provision_from_template --use-provider rhos7-ga}}

**Comment by dajoRH** [comment]
Lint report for commit 991c6a199819fdba6c689d66d752fb1899b76d10:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d3e8d1874517849db9c6b1f5b456588ec76b28a2:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**